spec/models/video_spec.rb

```diff
@@ -2,7 +2,44 @@
 
 describe Video do
 
-  it { should have_many(:categories)}
+  it {should have_many(:categories)}
   it {should validate_presence_of(:title)}
   it {should validate_presence_of(:description)}
-end
\ No newline at end of file
+  
+  describe "search_by_title" do
+            
+    it "returns empty array if search finds no video" do
+      Video.create([{title: 'South Park', description: 'Funny'}, 
+                             {title:'South Lawn', description: 'Not Funny'}])
+      expect(Video.search_by_title('hello')).to eq([])
+    end
+    
+    it "returns single video array if there is a single match" do
+      Video.create({title:'South Lawn', description: 'Not Funny'})
```

Comment:
I am not sure I understand why you use curly brackets inside the method. I mean, it works but its a bit uncommon in Ruby.